app/controllers/ht_institutions_controller.rb

```diff
@@ -1,15 +1,65 @@
 # frozen_string_literal: true
 
 class HTInstitutionsController < ApplicationController
-  before_action :fetch_institution, only: %i[show]
+  before_action :fetch_institution, only: %i[show edit update]
+
+  PERMITTED_UPDATE_FIELDS = %i[
+    grin_instance
+    name
```

Comment:
The bug with the US checkbox not doing anything is solved by adding `us` to this list. I note that once changed to from non-US to US, can't change it back in the UI. Is this by design?

Reply:
Probably not -- probably makes sense just to have it be fully editable. I think my initial thought was that it wasn't something that would change.

Reply:
Institutions seceding from the Union... hope not :-/
But, since add institution functionality is emerging, makes sense to treat the edit form like one would a fully editable new institution.